queries/bus.py

```diff
@@ -753,29 +762,16 @@ def sentence(state: QueryStateDict, result: Result) -> None:
         # Successfully matched a query type
         q.set_qtype(result.qtype)
         q.set_key(result.qkey)
-        # SQLAlchemy session, if required
-        session = state["session"]
         # Select a query function and execute it
         qfunc = _QFUNC.get(result.qtype)
-        answer: Optional[str] = None
-        if qfunc is None:
-            # Something weird going on - should not happen
-            answer = cast(str, result.qtype) + ": " + cast(str, result.qkey)
-            q.set_answer(dict(answer=answer), answer)
-        else:
-            try:
-                voice_answer = None
-                response: Union[AnswerTuple, ResponseType] = qfunc(q, session, result)
-                if isinstance(response, tuple):
-                    # We have both a normal and a voice answer
-                    response, answer, voice_answer = response
-                assert answer is not None
-                q.set_answer(response, answer, voice_answer)
-            except AssertionError:
+        try:
+            assert qfunc is not None, "qfunc is None"
+            q.set_answer(*qfunc(q, result))
+        except AssertionError:
+            raise
```

Comment:
Is this definitely what we want in production?

Reply:
Probably not, no. I didn't modify this in the pull request though. I guess I'll just remove the `except AssertionError: ...` thing for now.